src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -108,22 +98,16 @@ export const SchemaDrivenAnnotationEditor: React.FunctionComponent<SchemaDrivenA
 
   const ANNOTATION_EDITOR_TOOLTIP_ID = 'AnnotationEditorTooltip'
 
-  const { data: json } = useGetJson(entityId!, {
+  const { entityMetadata: entityJson, annotations } = useGetJson(entityId!, {
     enabled: !entityId || !formData, // once we have data, don't refetch. it would overwrite the user's entries
   })
 
   useEffect(() => {
-    if (json) {
-      /**
-       * To only show annotations in the form, we must remove non-annotation fields.
-       * We will need to submit these values when we create the entity, so we set them aside
-       * and will merge objects later.
-       */
-
-      setEntityJson(getStandardEntityFields(json))
-      setFormData(removeStandardEntityFields(json))
```

Comment:
Separating the data into annotation and non-annotation data is now done within the custom hook